### PR TITLE
New package: dfu-util

### PR DESCRIPTION
**Pull request by foxlet**

### CODE REVIEW

**Comment by cr6git** [comment]
You forgot the maintainer.

File: srcpkgs/dfu-util/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+pkgname=dfu-util
+version=0.9
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="libusb-devel"
+short_desc="USB Device Firmware Class programmer"
+maintainer="Foxlet <foxlet@furcode.co>"
+homepage="http://dfu-util.sourceforge.net/"
+license="GPL-2.0-only"
```
Comment (cr6git) [inline srcpkgs/dfu-util/template]: see the man page:
````
LICENCE
       dfu-util is covered by the GNU General Public License (GPL), version 2
       or later.
````

Comment (foxlet) [reply srcpkgs/dfu-util/template]: The website itself only mentions GPLv2, which is newer than when the manpage was written.

Comment (cr6git) [reply srcpkgs/dfu-util/template]: all right.

**Pull request merged**